plugins/plugin-react-refresh/test/plugin.test.js

```diff
@@ -95,4 +99,23 @@ describe('@snowpack/plugin-react-refresh', () => {
     );
     await testPluginInstance(pluginInstance);
   });
+
+  test('include custom babel config', async () => {
+    const babel = require('@babel/core');
+    const mockTransformAsync = jest.fn(() => Promise.resolve({code: ''}));
+    jest.spyOn(babel, 'transformAsync').mockImplementation(mockTransformAsync);
+
+    const babelConfig = {
+      compact: true,
+      plugins: ['@babel/plugin-syntax-top-level-await'],
+    };
+    const pluginInstance = pluginReactRefresh({}, {babel: babelConfig});
+    await pluginInstance.transform({...jsTransformOptions});
+
+    expect(mockTransformAsync).toHaveBeenCalledTimes(1);
+    expect(mockTransformAsync.mock.calls[0][1]).toHaveProperty('compact', true);
+    expect(mockTransformAsync.mock.calls[0][1].plugins).toContain(
+      '@babel/plugin-syntax-top-level-await',
+    );
+  });
```

Comment:
👍🏻 Thanks for adding a test!

Normally I prefer for tests to not mock too much, because it‘s too easy to miss breakages. But in this case I understand simply asserting that a plugin was passed to Babel